lib/View/homeScreen.dart

```diff
@@ -3,6 +3,7 @@ import 'package:qr_flutter/qr_flutter.dart';
 import 'package:intl/intl.dart';
 import 'dart:async';
 import 'package:screen_protector/screen_protector.dart';
+import 'package:deals_dray_test/Domain/services.dart';
 
```

Comment:
nit: to use function getValue (shared Preference) to get the students enrollment no.